torchgen/model.py

```diff
@@ -2594,7 +2611,7 @@ def __str__(self) -> str:
                 if self.functional_overload
                 else ""
             )
-            return f"{self.base}{i}"
+            return f"{self.namespace}{self.base}{i}"
```

Comment:
hmm does this mean the name of ops will show up as e.g. `atenadd`? Shouldn't we prefer e.g. `aten::add` instead? (including the colons if the namespace is specified)